### ISSUE TITLE
no valid error message if no tex file is givin

### ISSUE THREAD

**Issue by buhtz** [ftilmann/latexdiff#85]
I am using `latexdiff` (`This is LATEXDIFF 1.1.1  (Algorithm::Diff 1.15 fast, Perl v5.24.1)`) in Debian unstable.

When no texfile is given
e.g. `latexdiff`
e.g. `latexdiff --version`

There should be a valid and human readable error message about that.

Instead of that there is ugly code-intern error messages not usable for a user
```

Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\includeonly{ <-- HERE (.*?)}/ at /usr/bin/latexdiff line 1572.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/(^(?:[^%\n]|\\%)*)\\bibliography{ <-- HERE (.*?)}/ at /usr/bin/latexdiff line 1607.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/(^(?:[^%\n]|\\%)*)\\subfile{ <-- HERE (.*?)}/ at /usr/bin/latexdiff line 1622.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/(?<!\\)\\{ <-- HERE / at /usr/bin/latexdiff line 2351.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{ <-- HERE MATHMODE}((?:(.(?!(?<!\\)\&|\\\\))*)?)\\end{MATHMODE}/ at /usr/bin/latexdiff line 2676.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{MATHMODE}((?:(.(?!(?<!\\)\&|\\\\))*)?)\\end{ <-- HERE MATHMODE}/ at /usr/bin/latexdiff line 2676.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{ <-- HERE MATHMODE}(.*?)\\end{MATHMODE}/ at /usr/bin/latexdiff line 2678.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{MATHMODE}(.*?)\\end{ <-- HERE MATHMODE}/ at /usr/bin/latexdiff line 2678.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\end{ <-- HERE SQUAREBRACKET}/ at /usr/bin/latexdiff line 2759.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{ <-- HERE SQUAREBRACKET}/ at /usr/bin/latexdiff line 2760.

```

**Comment by ftilmann** [comment]
The problem with the unescaped 'left brace' warnings has been fixed in 1.2.0 , so you either install the current version of latexdiff separately, or you lobby with the Debian people to upgrade. In any case they are simply warnings, not error messages, and can for the time being be ignored. 

And you _should_ get a clear error message when running latexdiff without options:
```
> latexdiff
2 and only 2 non-option arguments required.  Write latexdiff -h to get help
```
If this is not the case I can only imagine that some special adaptions have been made for inclusion in debian (though I could not imagine what would be the reason for that, so please double-check your output/script).

**Comment by buhtz** [comment]
This answer is realy helpfull. Thanks.
Doesn't you have a red line to your debian maintainer? If not I will open a report.
Could you please specify the commit of the bugfix here.

kind
